File: Assignment12/program2.c
/////////////////////////////////////////////////////////////////// 
//
// File name : program2.c
// Description : Pattern
// input : iRow = 4  iCol = 3
// Output : 
/*
    1   2   3
    1   2   3
    1   2   3
    1   2   3
*/
// Author : Adinath Santosh Pawar
// Date : 27/5/25
// 
///////////////////////////////////////////////////////////////////

#include<stdio.h>

void Pattern(int iRow,int iCol)
{
    int j = 0;
    int i = 0;

    for(i = 1; i <= iRow; i++)
    {    
        for(j = 1; j <= iCol; j++)
        {
            printf("%d \t",j);
        }
        printf("\n");
    }

}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("enter number of rows \n");
    scanf("%d",&iValue1);

    printf("enter number of columns \n");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);
    return 0;
}

File: Assignment46/program2.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program2.java
// Description : Pattern printing
// Author : Adinath Santosh Pawar
// Date : 20/8/25
// 
///////////////////////////////////////////////////////////////////

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        for(int i = 1; i <= iRow; i++)
        {
            char chCap = 'A';
            char chSmall = 'a';
            for(int j = 1; j <= iCol; j++)
            {
                if(i % 2 == 0)
                {
                    System.out.print(chSmall + " ");
                    chSmall++;
                }
                else
                {
                    System.out.print(chCap + " ");
                    chCap++;
                }
                
            }
            System.out.println();
        }
    }
}
class program2
{
    public static void main(String A[])
    {
       Pattern pobj = new Pattern();
       Scanner sobj = new Scanner(System.in);

       System.out.print("Enter the number of rows ");
       int iRow = sobj.nextInt();

       System.out.print("Enter the number of columns ");
       int iCol = sobj.nextInt();
       pobj.Pattern(iRow,iCol);
    }
}